integration_tests/run_pyspark_from_build.sh

```diff
@@ -270,6 +270,10 @@ else
       fi
     fi
 
+    # Set a seed to be used for parallel jobs injecting OOMs in tests, as all tests names must
+    # match, and we use the test name to label whether OOM injection was attempted.
+    export SPARK_RAPIDS_TEST_INJECT_OOM_SEED=`date +%s`
```

Comment:
We need to print this out and have a way to override it so that we can reproduce failures.

Reply:
Note that if you pass: `--test_oom_injection_mode=always` to `./run_pyspark_from_build.sh` the whole seed part is completely skipped, and we just always inject.

But if there is a failure/leak due to the order of tests, then I am with you. I am going to echo it from this script, and provide a user-settable variable.

Reply:
Handled here: https://github.com/NVIDIA/spark-rapids/pull/7925/commits/c3e4b2c874e88dbd737f2d66bdce8655aa9829df